docs/readthedocs/source/doc/PPML/Overview/install.md

```diff
@@ -0,0 +1,54 @@
+# PPML Installation
+
+---
+
+#### OS requirement
+
+
+```eval_rst
+.. note::
+    **Hardware requirements**:
+
+     Intel SGX: PPML's features (except Homomorphic Encryption) are mainly built upon Intel SGX. Intel SGX requires Intel CPU with SGX feature, e.g., IceLake (3rd Xeon Platform). [Check if your CPU has SGX feature](https://www.intel.com/content/www/us/en/support/articles/000028173/processors.html).
+```
+```eval_rst
+.. note::
+    **Supported OS**:
+
+     PPML is thoroughly tested on Ubuntu (18.04/20.04), and should works fine on CentOS/Redhat 8.
+```
+
+#### Enable SGX for your Cluster
+
+```eval_rst
+.. mermaid::
+   
+   graphe TD
```

Comment:
Seems like here is a type. We should have `graph` instead of `graphe` for correct mermaid syntax